packages/frontend/src/components/ChartConfigPanel/Legend/ReferenceLine.tsx

```diff
@@ -1,26 +1,40 @@
 import { Button, FormGroup, InputGroup } from '@blueprintjs/core';
+import { DateInput2 } from '@blueprintjs/datetime2';
 import {
+    AdditionalMetric,
     CompiledDimension,
     Field,
     fieldId as getFieldId,
+    formatDate,
+    getDateFormat,
+    isAdditionalMetric,
+    isDateItem,
+    isDimension,
     isField,
     isNumericItem,
     TableCalculation,
+    TimeFrames,
+    WeekDay,
 } from '@lightdash/common';
 import debounce from 'lodash/debounce';
+import moment from 'moment';
 import { FC, useCallback, useMemo, useState } from 'react';
 import FieldAutoComplete from '../../common/Filters/FieldAutoComplete';
+import MonthAndYearInput from '../../common/MonthAndYearInput';
+import { ReferenceLineField } from '../../common/ReferenceLine';
 import { Flex } from '../../common/ResourceList/ResourceTable/ResourceTable.styles';
```

Comment:
I think we should duplicate this or make this flex a common component which will be reused both in ResourceTable and ReferenceLine